<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShowGRRDialog</class>
 <widget class="QDialog" name="ShowGRRDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>891</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gauge Repeatability and Reproducibility</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="MainHorizontalLayout">
     <item>
      <widget class="QTableWidget" name="GrrTableWidget"/>
     </item>
     <item>
      <widget class="QGroupBox" name="ItemListGroupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string>TestItems</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="DeselectHorizontalLayout">
          <item>
           <widget class="QPushButton" name="SelectAllButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Select All</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="UnSelectAllButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Deselect All</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <widget class="QListWidget" name="TestItemListWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="ShowItemHorizontalLayout">
     <item>
      <widget class="QPushButton" name="ShowFailItemButton">
       <property name="text">
        <string>Show Fail TestItems</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ShowAllItemsButton">
       <property name="text">
        <string>Show All TestItems</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="ShowItemHorizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="SaveResultButton">
       <property name="text">
        <string>Save Result</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="SelectItemButton">
       <property name="text">
        <string>Select TestItems</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="OkButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
